source/comply/compliance-monitoring.rst

```diff
@@ -3,7 +3,7 @@ Compliance monitoring
 
 .. This page is intentionally NOT accessible from the LHS.
 
-|enterprise| |self-hosted|
+|enterprise| |cloud| |self-hosted|
 
 .. |enterprise| image:: ../images/enterprise-badge.png
```

Comment:
@jasonblais - We'll need to add the inline image definition to the page for the Cloud badge to display correctly:

```
.. |cloud| image:: ../images/cloud-badge.png
  :scale: 30
  :target: https://mattermost.com/deploy
  :alt: Available for Mattermost Cloud deployments.
```

Reply:
Added @cwarnermm :+1: